### PR TITLE
Sort command line options

### PR DESCRIPTION
**Pull request by nik9000**
This tidies the list of command line options so they are always in
alphabetical order within their three groups.

### CODE REVIEW

File: build_docs.pl [hunk 1]
```diff
@@ -680,32 +673,6 @@ sub init_env {
         or die "Please install <xsltproc>";
 }
 
-#===================================
```
Comment (nik9000) [inline build_docs.pl]: I moved this so it'd be by its buddies.

Comment (nik9000) [reply build_docs.pl]: And I sorted it.

Comment (jarpy) [reply build_docs.pl]: :+1: Command-line options should stick together in these troubled times.

**Comment by nik9000** [comment]
Thanks for reviewing @jarpy!

**Pull request merged**